flask_exceptions/extension.py

```diff
@@ -125,41 +125,41 @@ def init_app(self, app, config=None):
     # Exception class wrappers sorted by error code
 
     @exception('Invalid request parameters')
-    def bad_request(self, **kwargs):
+    def bad_request(self, *_, **kwargs):
```

Comment:
Since the decorator moves the `message` arg to a kwarg, pylint complains that this function doesn't accept args. Well now it does - they're just not used.